test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
Interesting, didn't know about inject-then. I'd be interested to see how this scales — what do you think the value of having two separate servers is?